### ISSUE TITLE
Why am I getting different learning curve for SVC in using sklearn and sklearnex

### ISSUE THREAD

**Issue by YuanHank** [intel/scikit-learn-intelex#1390]
&emsp;Now I'm using SVM.SVC to train the data.Due to the long execution time for using sklearn,I try to implement the sklearnex.After using same datas and parameters below, I get two different learning curves.Wondering if anyone run into the same issue,if so, how to due with it or where't the problem.

parameters:
* kernel = 'rbf'
* C = 1
* gamma = 1e-2
* max_iter = 3000
* random_state = 42
* tol =1e-3
* probability=True
* cache_size =500

packages version:
* scikit-learn - 1.1.1
* scikit-learn-intelex - 2023.2.1

learning curves:
* [By sklearn,with five fold](https://i.stack.imgur.com/qLmQS.png)
* [By sklearnex,with five fold](https://i.stack.imgur.com/VMo8E.png)

**Comment by Alexsandruss** [comment]
sklearn and sklearnex implementations of SVM are expected to show different results.
What score is used as loss in these learning curves?

**Comment by YuanHank** [comment]
In here I used f1 score as the matric, the label of the y-axis might be wrong. So which result should I trust,or both are correct?

**Comment by Alexsandruss** [comment]
Formally, flattened learning curve for training/validation and higher f1-score for sklearnex is not an issue. It looks like sklearn has convergence issue for bigger number of samples.

**Comment by syakov-intel** [comment]
Score issue is not an issue in this case.

**Comment by napetrov** [comment]
Reference back to stackoverflow post - https://stackoverflow.com/questions/76842446/why-am-i-getting-different-learning-curve-for-svc-in-using-sklearn-and-sklearnex and answer from there

"I ran a somewhat similar experiment, and got consistent results between sklearn/sklearnex:
![image](https://github.com/intel/scikit-learn-intelex/assets/18015221/7f1fbafc-770b-4a3a-a3be-a8e9b2ffd1e3)

I was using SVR with 5-fold CV and a similarly-sized dataset to yours. I ran it under sklearn 1.3.0. and sklearnex 2023.2.1
"
